android/src/main/java/com/google/samples/apps/ibeaconapp/lightbluebean/Beacon.java

```diff
@@ -9,11 +9,13 @@
     private Bean bean;
     private int rssi;
     private String roomName;
+    private String roomId;
 
-    public Beacon(Bean bean, int rssi, String roomName) {
+    public Beacon(Bean bean, int rssi, String roomName, String roomId) {
```

Comment:
https://issues.griddynamics.net/browse/RND-104
says "Room with iBeacon",
but here ibeacon is associated with room. just for note